screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -704,7 +705,7 @@ class EditTransactionViewModel @Inject constructor(
                     dateTime = when {
                         loadedTransaction().dateTime == null &&
                                 dueDate.value == null -> {
-                            timeNowUTC()
+                            timeProvider.utcNow().atZone(timeProvider.getZoneId()).toLocalDateTime()
```

Comment:
I tested it, the date is showing correctly after saving the transaction but the is issue in the balance calculation. the balance didnt get updated for me...

Reply:
Thank you @akashs056! Much appreciated that you tested 🙌

Reply:
@akashs056 thanks for pointing it out.